server/models/household.js

```diff
@@ -1,9 +1,12 @@
 let mongoose = require('mongoose');
-let Student = require('./student').schema;
+
+let Student = require('./student')
+
+
 let Schema = mongoose.Schema;
 
 let HouseholdSchema = new Schema({
-    students: [Student],
+    students: [Object],
```

Comment:
Is this supposed to happen?

Reply:
Yeah because mongoose has a weird thing with specifying the object type, we should leave it as an object in the schema